db/data/test-data/index.js

```diff
@@ -2,3 +2,5 @@ exports.categoryData = require('./categories.js');
 exports.commentData = require('./comments.js');
 exports.reviewData = require('./reviews.js');
 exports.userData = require('./users.js');
+
+module.exports = { categoryData, reviewData, userData, commentData}
```

Comment:
please remove this as it is already done by default using the dot notation :) - there is no need to amend this file